boards/arm/stm32/nucleo-f302r8/configs/qenco/defconfig

```diff
@@ -17,24 +71,40 @@ CONFIG_ARCH_INTERRUPTSTACK=1024
 CONFIG_ARCH_IRQPRIO=y
 CONFIG_BOARD_LOOPSPERMSEC=8499
 CONFIG_BUILTIN=y
-CONFIG_DEBUG_FULLOPT=y
```

Comment:
@xiaoxiang781216 please correct me if I'm wrong, but isn't full opt the same as Os?

Reply:
Let's get back from custom opt to full opt

Reply:
Done 👍🏼